.github/workflows/check_pr_plugin_aws.yml

```diff
@@ -82,6 +82,6 @@ jobs:
           export API_TOKEN="${{ secrets.API_TOKEN }}"
           export SPACES_KEY="${{ secrets.SPACES_KEY }}"
           export SPACES_SECRET="${{ secrets.SPACES_SECRET }}"
-          export AWS_ACCESS_KEY_ID="${{ secrets.S3_RESOTOPUBLIC_AWS_ACCESS_KEY_ID }}"
-          export AWS_SECRET_ACCESS_KEY="${{ secrets.S3_RESOTOPUBLIC_AWS_SECRET_ACCESS_KEY }}"
-          awspolicygen --verbose --spaces-name somecdn --spaces-region ams3 --spaces-path resoto/aws/ --aws-s3-bucket resotopublic --aws-s3-bucket-path cf/
+          export AWS_ACCESS_KEY_ID="${{ secrets.S3_FIXPUBLIC_AWS_ACCESS_KEY_ID }}"
+          export AWS_SECRET_ACCESS_KEY="${{ secrets.S3_FIXPUBLIC_AWS_SECRET_ACCESS_KEY }}"
+          awspolicygen --verbose --spaces-name somecdn --spaces-region ams3 --spaces-path fix/aws/ --aws-s3-bucket fixpublic --aws-s3-bucket-path cf/
```

Comment:
let's make this path fixinventory/aws/ since the bucket `fixpublic` already exists and hosts the public Fix SaaS CF templates.

Reply:
Ok created a new bucket `fixinventorypublic` and corresponding credentials in `S3_FIXINVENTORYPUBLIC_AWS_ACCESS_KEY_ID` and `S3_FIXINVENTORYPUBLIC_AWS_SECRET_ACCESS_KEY`